README.md.txt

```diff
@@ -0,0 +1,84 @@
+# Collaborative Create-Your-Own-Adventure
+
+This is a create-your-own-adventure story where at each step, the reader chooses
+what action to take and turns to an appropriate page of the story based on their
+choice.  It is collaborative because it is written one small piece at a time by
+many authors.  To start reading the story, click [here](language.md).
+
+## Format of the Story
+
+Each "page" of this story is a separate file.  The files are Markdown, which
+means they are plain text, but with a few special capabilities.  The only
+Markdown capability used in this story is links.  By putting text in brackets
+and a URL in parentheses, you can add a link to the story.  For example, the
+following text would turn "here" into a link to Google:
+
+    Click [here](http://www.google.com) to visit Google.
+
+## Contribution Guidelines
+
+### Make Appropriately Sized Changes
+
+Try to keep your change small.  Part of the fun of a collaborative story is
+that no one person writes very much of it!  On the other hand, please **don't**
+add an option without adding any story for readers who choose that option, or
+the story will soon be full of dead links.  Instead, the last `.md` file the reader
+reaches on any path of the story should not have any options.  For an example, see
+[this file](https://github.com/udacity/create-your-own-adventure/blob/c4f2bf7caac641df1c979a2db056a532e0c93e23/english/light-fire/fire.md)
```

Comment:
This link seems to be an absolute link to GitHub rather than a relative link.  If you are linking to another page in the create-your-own-adventure repository, please use relative links to do so.  See the README of this repository for more information.